src/Facades/Image.php

```diff
@@ -7,13 +7,13 @@
 use Illuminate\Support\Facades\Facade;
 
 /**
- * @method \Intervention\Image\ImageManager static read(mixed $input, string|array|DecoderInterface $decoders = [])
- * @method \Intervention\Image\ImageManager static create(int $width, int $height)
- * @method \Intervention\Image\ImageManager static animate(callable $callback)
+ * @method static \Intervention\Image\Interfaces\ImageInterface read(mixed $input, string|array|\Intervention\Image\Interfaces\DecoderInterface $decoders = [])
+ * @method static \Intervention\Image\Interfaces\ImageInterface create(int $width, int $height)
+ * @method static \Intervention\Image\Interfaces\ImageInterface animate(callable $callback)
  */
 class Image extends Facade
 {
-    protected static function getFacadeAccessor()
+    protected static function getFacadeAccessor(): string
```

Comment:
Doesn't that create an incompatibility with `Illuminate\Support\Facades\Facade::getFacadeAccessor()`?

Reply:
This shouldn't be a problem, as the `Illuminate\Support\Facades\Facade::getFacadeAccessor()` return type is inferred from PHPDoc. However, I've removed it just to be safe.